q2cli/commands.py

```diff
@@ -336,6 +348,21 @@ def __call__(self, **kwargs):
             raise ValueError('Cannot set a pool to be used for recycling and '
                              'no recycle simultaneously.')
 
+        parsl = kwargs.pop('parsl', False)
+        parsl_config_fp = kwargs.pop('parsl_config', None)
+
+        if parsl and parsl_config_fp is not None:
+            raise ValueError('Cannot use both --parsl and --parsl-config. '
```

Comment:
Let's make it so that `--parsl-config` is an optional extra that overrides the default config.